CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 
 - Minor: The `redirect_uri` config option under the `[twitchapi]` section is now optional with a sane default value: `https://<domain>/login/authorized`. (#1618)
 - Minor: Allow configuring streamer, bot, admin, and control hub using Twitch User IDs instead of Twitch User Logins. **Specifying user logins in the config has therefore now been deprecated and may stop working in a future release.** (#1590)
+- Minor: Add enable/disable command functionality to `!edit`. (#1499)
```

Comment:
PR numbers should be used instead of issue numbers in changelog entries.
In my opinion, the changelog entry (as well as message returned to the user) is misleading and doesn't explain the functionality well since `--enable` and `--disable` arguments already exist. 

```suggestion
- Minor: Add enable/disable command functionality to `!edit`. (#1678)
```

This PR does not disable the command but rather "disables the command from chat", which isn't very clear and should be addressed properly, changing messages returned to the user - see my other comments.